src/nodeDebugAdapter.ts

```diff
@@ -457,23 +457,26 @@ export class NodeDebugAdapter extends ChromeDebugAdapter {
         }
     }
 
-    public configurationDone(): Promise<void> {
+    public async configurationDone(): Promise<void> {
         if (!this.chrome) {
             // It's possible to get this request after we've detached, see #21973
-            return super.configurationDone();
+            await super.configurationDone();
+            return;
         }
 
         // This message means that all breakpoints have been set by the client. We should be paused at this point.
         // So tell the target to continue, or tell the client that we paused, as needed
         this._finishedConfig = true;
         if (this._continueAfterConfigDone) {
             this._expectingStopReason = undefined;
-            this.continue(/*internal=*/true);
+            await this.continue(/*internal=*/true);
         } else if (this._entryPauseEvent) {
-            this.onPaused(this._entryPauseEvent);
+            await this.onPaused(this._entryPauseEvent);
         }
 
-        return super.configurationDone();
+        await super.configurationDone();
+
+        this.events.emit(ChromeDebugSession.FinishedStartingUpEventName);
```

Comment:
Does this have to go after the super call? Would rather leave the super call as the last line unless there's a reason for it.

Reply:
Changed